virtcontainers/kata_agent.go

```diff
@@ -1708,3 +1726,39 @@ func (k *kataAgent) convertToRoutes(aRoutes []*aTypes.Route) (routes []*types.Ro
 
 	return routes
 }
+
+func (k *kataAgent) copyFile(src, dst string) error {
+	var st unix.Stat_t
+
+	err := unix.Stat(src, &st)
+	if err != nil {
+		return fmt.Errorf("Could get file %s information", src)
+	}
+
+	b, err := ioutil.ReadFile(src)
+	if err != nil {
+		return fmt.Errorf("Could not read file %s", src)
```

Comment:
Again, it would help to include the error so the user can see why the read failed.

Reply:
yes, thanks :smile: